CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+## 0.0.1-dev
+
+* Initial release
+
+## 0.0.2-dev
```

Comment:
Put the versions in a descending order as (I believe) the standard is the descending order.

Reply:
Fixed